README.md

```diff
@@ -130,6 +137,24 @@ Furthermore the `PYTHONPATH` needs to include the `pythonlibs` of RIOT:
 export PYTHONPATH=${RIOTBASE}/dist/pythonlibs:${PYTHONPATH}
 ```
 
+The environment variable `RIOTBASE` must be set to *absolute path* of the
+version of RIOT under test. E.g.
+
+```
+export RIOTBASE=$(readlink -f ../RIOT)
+```
+
+Some tests on the `native` platform need a certain number of TAP interfaces in a
+bridge or otherwise will be skipped. The most number of TAP interfaces to date
+is required for 3.5 "ICMPv6 stress test on native (neighbor cache
+stress)" (11 TAP interfaces) so to not skip that, all of them should be bridged.
+
+```sh
+sudo ${RIOTBASE}/dist/tools/tapsetup/tapsetup -c 11
+```
+
+#### Iotlab Requirements
+
 By default experiments will be launched on [IoT-LAB saclay site] since it has
 most of the boards used in the release-specs.
```

Comment:
This sentence is a repetition of the introduction. Maybe word it differently to lead into the following sentence or just remove the reasoning.

Reply:
Ok I removed, let me know if things are ok now.

Reply:
:+1: